src/js/visual/configurator.js

```diff
@@ -912,6 +914,7 @@ ripe.Configurator.prototype._loadFrame = async function(view, position, options
         if (isReady) {
             await this._drawFrame(image, animate, duration);
         }
+        this.trigger("post_frame", view, position, options, true);

```

Comment:
Aren't these `true`/`false` the opposite of what they are in `post_update`?
- True: updated happened
- False: "cache" hit / no-op